src/lib.rs

```diff
@@ -45,6 +72,34 @@ pub struct BMFont {
 }
 
 impl BMFont {
+    /// Constructs a new [BMFont].
+    ///
+    /// ## Examples
```

Comment:
I think it should be ###, not sure though.

Reply:
I was using ## from some other code base as an example; but now that I look at [the Rust source](https://doc.rust-lang.org/src/core/iter/traits/iterator.rs.html#111) it looks like # might be better in this situation.

I'll change it to that - feel free to modify/say or we let someone else provide substantial docs?

Reply:
Agree with #.